src/__tests__/CompareResults/ResultsView.test.tsx

```diff
@@ -150,4 +150,53 @@ describe('Results View', () => {
     expect(createObjectURLMock).toHaveBeenCalled();
     expect(revokeObjectURLMock).toHaveBeenCalledWith('blob:');
   });
+
+  it('click on retrigger button', async () => {
+    const user = userEvent.setup({ delay: null });
+
+    const { testCompareDataWithMultipleRuns, testData } = getTestData();
+    (window.fetch as FetchMockSandbox)
+      .get(
+        'begin:https://treeherder.mozilla.org/api/perfcompare/results/',
+        testCompareDataWithMultipleRuns,
+      )
+      .get('begin:https://treeherder.mozilla.org/api/project/', {
+        results: [testData[0]],
+      });
+
+    const mockGetRandomValues = jest.fn().mockReturnValue(new Uint32Array(1));
+    Object.defineProperty(window, 'crypto', {
+      value: { getRandomValues: mockGetRandomValues },
+    });
```

Comment:
I believe we can use [Node version of window.crypto](https://nodejs.org/docs/latest-v18.x/api/webcrypto.html):

```js
import { webcrypto } from 'node:crypto';

...

Object.defineProperty(window, 'crypto', value: webcrypto);
```
(see https://github.com/jsdom/jsdom/issues/1612#issuecomment-1659591074 as well)

I think you can add this to `setupTests.ts` so that this is available for all tests.